README.md

```diff
@@ -63,6 +63,29 @@ The ClearHandler function is provided by the gorilla/context package.
 More examples are available [on the Gorilla
 website](http://www.gorillatoolkit.org/pkg/sessions).
 
+## Multiple Flashes Example
+For getting more than one flashes in a map you can use below method in your code :
```

Comment:
"For retrieving multiple flashes from a single session, you can loop over the Flashes map. Make sure to call `session.Save(r, w)` so that the changes are persisted to the session."